pkg/pipelines/pipelines.go

```diff
@@ -398,3 +400,56 @@ func GetLatestPipelinerun(c *clients.Clients, namespace string) (string, error)
 	return prs.Items[0].Name, nil
 
 }
+
+func CheckLogVersion(c *clients.Clients, binary, namespace string) {
+	prname, err := GetLatestPipelinerun(store.Clients(), store.Namespace())
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun: %v", err))
+		return
+	}
+	// Get PipelineRun logs
+	logsBuffer, err := getPipelinerunLogs(c, prname, namespace)
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun logs: %v", err))
+		return
+	}
+	logs := logsBuffer.String()
```

Comment:
Why don't you call this function on line 411 and spare one variable?

Reply:
Done